pkg/api/v1/conversion_test.go

```diff
@@ -23,28 +23,28 @@ import (
 	versioned "k8s.io/kubernetes/pkg/api/v1"
 )
 
-func TestNodeConversion(t *testing.T) {
-	obj, err := versioned.Codec.Decode([]byte(`{"kind":"Minion","apiVersion":"v1"}`))
-	if err != nil {
-		t.Fatalf("unexpected error: %v", err)
-	}
-	if _, ok := obj.(*api.Node); !ok {
-		t.Errorf("unexpected type: %#v", obj)
-	}
+//func TestNodeConversion(t *testing.T) {
```

Comment:
Please delete instead of commenting out. (git keeps a record should we ever need it again)